gnovm/pkg/gnolang/gno_test.go

```diff
@@ -245,6 +245,38 @@ func main() {
 	m.RunMain()
 }
 
+func BenchmarkForLoop(b *testing.B) {
```

Comment:
What do you think of having these simply as part of the benchdata tests?

There is a `loop.gno` test already, there's one missing for loop+if that you added. You can also run them individually:

```
go test -bench Benchdata/loop -run=^$ ./gnovm/pkg/gnolang/
```

Reply:
Nevermind, I understand now the benchmarks are about preprocessing, not execution. Can you clarify that in the name? Ie. `BenchmarkPreprocessForLoop`

Reply:
Ok